components/infobox/wikis/counterstrike/infobox_league_custom.lua

```diff
@@ -103,18 +97,18 @@ local RESTRICTIONS = {
 
 local _DATE_TBA = 'tba'
 
-local _TIER_VALVE_MAJOR = 'major'
-
 local _MODE_1v1 = '1v1'
 local _MODE_TEAM = 'team'
 
 local PRIZE_POOL_ROUND_PRECISION = 2
 
 local _args
+local _game
 
 function CustomLeague.run(frame)
 	local league = League(frame)
 	_args = league.args
+	_game = Game.raw{game = _args.game, useDefault = false}
```

Comment:
technically could just use `args.gameData = ...` to get rid of the "global" `_game` **if wanted**

Reply:
I prefer to avoid static variables as much as possible. For instance with the current version if two of these would be instantiated from a single invoke call, they would start sharing game information (`_game`). Obviously there are other issues with it as well (`_args`), but it's one less at least.